tests/scripts/components-platform.sh

```diff
@@ -299,12 +299,6 @@ component_build_sha_armce () {
 
 
     # test the deprecated form of the config option
-    scripts/config.py set MBEDTLS_SHA256_USE_A64_CRYPTO_ONLY
```

Comment:
I think the comment at line 301 (just above this block that you removed) should be deleted as well because it seems to reference the removed part.